x-pack/plugins/secops/public/store/local/reducer.ts

```diff
@@ -18,6 +19,7 @@ export interface LocalState {
   timeline: TimelineState;
   hosts: HostsState;
   inputs: InputsState;
+  errors: ErrorState;
```

Comment:
I would have put the errorState inside of app. No need to create more reducer and state for just one object